Src/Base/AMReX_OpenMP.H

```diff
@@ -13,7 +15,11 @@ namespace amrex::OpenMP {
     inline int in_parallel     () { return omp_in_parallel();     }
     inline void set_num_threads (int num) { omp_set_num_threads(num); }
 
-    void init_threads ();
+    void Initialize ();
+    void Finalize ();
+
+    static constexpr int nlocks = 128;
```

Comment:
A short inline comment what the relation between `nlocks` and number of OMP threads is (constraints or assumptions or performance impact) would be good here, I think.